site/app/templates/admin/UploadConfigForm.twig

```diff
@@ -35,30 +34,47 @@
     <br>
     <b>located in {{ target_dir }}</b>
     <br>&nbsp;<br>
+    <link>
+    <script>
+        function renameConfig(file_path){
```

Comment:
Since this function doesn't actually rename the config, maybe rename this to `openRenamePopup` or something that indicates that it will present the user with the option to rename.